src/gui/editors/PianoRoll.cpp

```diff
@@ -4002,10 +4039,8 @@ void PianoRoll::enterValue( NoteVector* nv )
 		bool ok;
 		int new_val;
 		new_val = QInputDialog::getInt(	this, "Piano roll: note velocity",
-					tr( "Please enter a new value between %1 and %2:" ).
-						arg( MinVolume ).arg( MaxVolume ),
-					(*nv)[0]->getVolume(),
-					MinVolume, MaxVolume, 1, &ok );
+		tr( "Please enter a new value between %1 and %2:" ).arg( MinVolume ).arg( 
+            MaxVolume ),(*nv)[0]->getVolume(),MinVolume, MaxVolume, 1, &ok );
```

Comment:
Could you revert this change? The old formatting is consistent with typical C++ formatting guidelines.